app/src/components/Header.Vue

```diff
@@ -0,0 +1,19 @@
+<template>
+    <div>
+        <h1>
+            hello world
+        </h1>
+        <RouterLink class="home" to="/"> Home </RouterLink>
```

Comment:
```suggestion
<RouterLink class="home" to="/"> Home </RouterLink>
<RouterLink class="Add" to="/grapplers"> Add Page </RouterLink>
```
Make sure tags are properly indented.